manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DimisAIO.DindeGDPS
+PackageVersion: 2024.01.26.0
+PackageLocale: en-US
+Publisher: DimisAIO
+PackageName: DindeGDPS
+License: None
+Copyright: Copyright (C) 2024 DimisAIO
+ShortDescription: DindeGDPS Installer

```

Comment:
Please provide a better description of what this package is, does, or is used for

Reply:
I hope I updated it right, never did this before...